imaging_tracking/admin.py

```diff
@@ -116,6 +118,24 @@ class ProfileInline(admin.StackedInline):
 
 class CustomUserAdmin(UserAdmin):
     inlines = [ProfileInline]
+    actions = ["make_owner_on_all_projects", "make_viewer_on_all_projects", "remove_all_roles"]
```

Comment:
Is the annotation of the returned type (`-> None`) needed for all the methods in actions or it's understood that they don't return anything because they are actions?

Reply:
I added type annotations